lib/include/openturns/OTgsobolusecase.hxx

```diff
@@ -0,0 +1,105 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The header file that defines the Ishigami Use Case
+ *
+ *  Copyright 2005-2024 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_OTISHIGAMIUSECASE_HXX
+#define OPENTURNS_OTISHIGAMIUSECASE_HXX
+
+#include "openturns/OTconfig.hxx"
+#include "openturns/Distribution.hxx"
+#include "openturns/SymbolicFunction.hxx"
+#include "openturns/Function.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+class GSobolUseCase
```

Comment:
the file name should take the name of the class

Reply:
and the macro protection uses the wrong name:
```
#ifndef OPENTURNS_OTISHIGAMIUSECASE_HXX
#define OPENTURNS_OTISHIGAMIUSECASE_HXX
```
should be
```
#ifndef OPENTURNS_OTGSOBOLUSECASE_HXX
#define OPENTURNS_OTGSOBOLUSECASE_HXX
```

Reply:
Ok. Fixed at 7fa306b985d1ebc27089a944161a2556366e27c7